app/src/main/java/com/andrew/tasky/presentation/event_detail/EventDetailFragment.kt

```diff
@@ -106,455 +98,402 @@ class EventDetailFragment : Fragment(R.layout.fragment_event_detail) {
+                        EventDetailViewModel.AttendeeFilterTypes.NOT_GOING -> {
+                            goingTextView.isVisible = false
+                            goingAttendeeRecyclerView.isVisible = false
+                            notGoingTextView.isVisible = true
+                            notGoingAttendeeRecyclerView.isVisible = true
+                            allButtonTextView.setTextColor(
+                                ResourcesCompat
+                                    .getColor(resources, R.color.dark_gray, null)
+                            )
+                            allButton.background.setTint(
+                                ResourcesCompat
+                                    .getColor(resources, R.color.light_2, null)
+                            )
+                            goingButtonTextView.setTextColor(
+                                ResourcesCompat
+                                    .getColor(resources, R.color.dark_gray, null)
+                            )
+                            goingButton.background.setTint(
+                                ResourcesCompat
+                                    .getColor(resources, R.color.light_2, null)
+                            )
+                            notGoingButtonTextView.setTextColor(
+                                ResourcesCompat
+                                    .getColor(resources, R.color.white, null)
+                            )
+                            notGoingButton.background.setTint(
+                                ResourcesCompat
+                                    .getColor(resources, R.color.black, null)
+                            )
+                        }
                     }
                 }
+                collectLatestLifecycleFlow(viewModel.goingAttendees) { goingAttendees ->
+                    val goingAttendeeAdapter = AttendeeItemAdapter(
+                        goingAttendees,
+                        isAttendee,
+                        onDeleteIconClick = { attendee -> viewModel.deleteAttendee(attendee) }
+                    )
+                    goingAttendeeRecyclerView.adapter = goingAttendeeAdapter
+                    goingAttendeeRecyclerView.layoutManager = LinearLayoutManager(
+                        requireContext()
+                    )
+                }
+                collectLatestLifecycleFlow(viewModel.notGoingAttendees) { notGoingAttendees ->
+                    val notGoingAttendeeAdapter = AttendeeItemAdapter(
+                        notGoingAttendees,
+                        isAttendee,
+                        onDeleteIconClick = { attendee -> viewModel.deleteAttendee(attendee) }
+                    )
+                    notGoingAttendeeRecyclerView.adapter = notGoingAttendeeAdapter
+                    notGoingAttendeeRecyclerView.layoutManager = LinearLayoutManager(
+                        requireContext()
+                    )
+                }
```

Comment:
Not sure if you already tried to apply the suggestion with only updating the list, but if so, this doesn't look like that, since you're still recreating the whole adapter and layout manager on every list change 😁

Reply:
It's on the todo list! :) Just haven't updated it yet!